fsm/join_add_csync.go

```diff
@@ -134,6 +134,12 @@ func JoinAddCsyncAction(z *music.Zone) bool {
 
 func VerifyCsyncPublished(z *music.Zone) bool {
 	log.Printf("Verifying Publication of CSYNC record sets for %s", z.Name)
+
+	if z.ZoneType == "debug" {
+		log.Printf("VerifyCsyncPublished: zone %s (DEBUG) is automatically ok", z.Name)
+		return true
+	}
```

Comment:
I am not a fan of this "debug" mode, but I don't have an intelligent suggestion for a better way of doing it. I understand the idea and the purpose. Just not a fan.

Reply:
Once we have a sufficiently robust system I suggest we rip it out. But it has been crucial and is still needed.